pkg/issuer/acme/dns/cloudflare/cloudflare.go

```diff
@@ -28,6 +28,9 @@ import (
 // TODO: Unexport?
 const CloudFlareAPIURL = "https://api.cloudflare.com/client/v4"
 
+// cloudFlareMaxBodySize the max size of the response body
+const cloudFlareMaxBodySize = 1024 * 1024 // 1mb
```

Comment:
suggestion: If I understand correctly from the discussion with @inteon  this is arbitrary. If it is this would benefit from a comment so that future maintainers know it can be grown or maybe shrunk without too much pain.

```suggestion
// cloudFlareMaxBodySize is the max size of a received response body. The value is arbitrary
// and is chosen to be large enough that any reasonable response would fit.
const cloudFlareMaxBodySize = 1024 * 1024 // 1mb
```